bin/code_climate_reek

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env ruby
+#
+# Wrapper for the CodeClimate integration.
+
+require_relative '../lib/reek'
+require_relative '../lib/reek/cli/application'
+
+# Map input coming from CodeClimate to Reek.
+class CodeClimate2Reek
```

Comment:
Splendid. The world is better now, if only a b̶i̶t̶ byte. (I’ll show myself out.)